SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -1,32 +1,47 @@
 <%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
 
-# How to port a native Unreal project to the Unreal GDK
-
-> This workflow is very subject to change. We're aware that it's not optimized, and one of our priorities is to improve iteration times.
+# How to port a native Unreal project to the SpatialOS GDK for Unreal
 
 ## Prerequisites
 
 <%(TOC)%>
 
-Follow the [setup guide]({{urlRoot}}/setup-and-installing) to the end of **Setting up the Unreal GDK module and Starter Project** > **Cloning**.
-> To speed up build times, install IncrediBuild, FastBuild, or another build distributor.
-
-## Setting up the project structure
-1. In a terminal window, navigate to the root directory of the SpatialOS Unreal GDK repository you cloned and run `Setup.bat`. This requires authorization with your SpatialOS account.
-1. Open the `StarterProject` repo and copy everything except the `\Game\` folder alongside your `<GameRoot>` folder. 
-Your folder structure should now resemble -
-```
-\<ProjectRoot>\ci\
-\<ProjectRoot>\<GameRoot>\
-\<ProjectRoot>\spatial\
-\<ProjectRoot>\ProjectPaths.bat
-\<ProjectRoot>\LaunchClient.bat 
-etc...
-```
-1. Open **`ProjectPaths.bat`** for editing and replace `Game` with `<GameRoot>`, and `StarterProject` with the name of your uproject (which we'll refer to as `<YourProject>`).
-
-## Adding the SpatialGDK module to your project
-1. In your project's `*.build.cs` file, add "SpatialGDK" to the `PublicDependencyModuleNames`.
+Before porting your project, you need to install and set up the SpatialOS GDK: follow the [set up guide]({{urlRoot}}/setup-and-installing) to the end of **Setting up the Unreal GDK module and Starter Project** > **Cloning**.
+
+## Terms used in this guide
+`<ProjectRoot>` - The root folder of your Unreal project.  
+`<GameRoot>` - The folder containing your game's `.uproject` and source folder (for example, `<ProjectRoot>/ShooterGame/`).  
+`<YourProject>` - Name of your game's `.uproject` (for example, `StarterProject.uproject`).
+
+## Set up the project structure
```

Comment:
Set up the project structure

->???

 Set up the game project structure

Reply:
To me this doesn't make much sense since the `game project` is the `GameRoot` in my eyes.
The `project` is the the folder containing the `GameRoot` since that has the spatial elements and whatever else the dev may need.

Reply:
We are flipping between "project" and "game" all over the docs and when you are reading this, the flipping between makes you think "what does this mean?" - it seems a little thing but changing terms are unsettling and confusing when you are in a strange new world and trying to make sense of things. It means you have to read things twice and you don't want to as there's enough to learn already. 
We should be consistent, at least in the same set of instructions (that is in this porting doc) even if we can't do it throughout all the docs. We should use the same term, unless there are occasions where it makes sense to use the different one. 
Because the terms are flipping all over the docs, I am suggesting using "game project" here as a catch-all as it's so inconsistent everywhere, but to be honest, I don't care which we use, as long as we don't keep switching. 
To me, it makes most sense to say "project" everywhere in this doc as we are dealing with files and talking about Unreal projects, apart from where we are talking about launching your game.